README.md

```diff
@@ -14,5 +14,22 @@ In order to use our tool, you have to install it in your Squeak image. Our prefe
 - For later updates, you can just pull the new version from the Git Repository. To do that, select the `dev` branch and click the button <kbd>Pull</kbd>.
 - Once the new version was downloaded, right click the latest commit and choose <kbd>Checkout objects</kbd>.
 
+### Metacello
+- Open a workspace and execute
+```smalltalk
+Metacello new
+    baseline: 'ChangesReloaded';
+    repository: 'github://hpi-swa-teaching/ChangesBrowser:dev/packages';
+    load
+```
+as well as
```

Comment:
```suggestion
and after that
```

Reply:
The order actually doesn't matter, so I would leave it the way it is...